samples/replicatedentity-counter/src/main/java/com/example/counter/domain/Counter.java

```diff
@@ -12,11 +12,6 @@ public Counter(ReplicatedEntityContext context) {
     this.entityId = context.entityId();
   }
 
-  @Override
-  public ReplicatedCounter emptyData(ReplicatedDataFactory factory) {
-    return factory.newCounter();
```

Comment:
Simpler with default empty data now provided by the specialised entity classes. Only the register will require an empty value now.